librato-iOS/Categories/NSString+AYLSanitizedForMetric.m

```diff
@@ -0,0 +1,21 @@
+//
+//  NSString+AYLSanitizedForMetric.m
+//  Librato-iOS
+//
+//  Created by Adam Yanalunas on 10/3/13.
+//  Copyright (c) 2013 Amco International Education Services, LLC. All rights reserved.
+//
+
+#import "NSString+AYLSanitizedForMetric.h"
+
+@implementation NSString (AYLSanitizedForMetric)
+
+- (NSString *)ayl_sanitizedForMetric
```

Comment:
I tried to avoid two letter prefix. AYL was choosen by your initials and L for librato. But can rename to any other.